### ISSUE TITLE
Crashes at lunch on Heroku.

### ISSUE THREAD

**Issue by victorferreira** [netlify/gocommerce#170]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
App crashes on first launch.

```
2019-01-29T11:30:52.336109+00:00 app[web.1]: time="2019-01-29T11:30:52Z" level=fatal msg="Error loading instance config: No payment providers enabled
```

**- If the current behavior is a bug, please provide the steps to reproduce.**
Deploy a new application to Heroku using the "Deploy to Heroku" button.

**- What is the expected behavior?**
App should lunch after deployed.

**- Please mention your Go version, and operating system version.**
Just using the standard Heroku buildpack.

I know it says the a I don't have payment provider enabled on the log, but here's a few things to consider:

- The README doesn't list the Payment section as `required`
- Even after adding `PAYMENT_STRIPE_ENABLED=true` and restarting the server, the app still does't launch.

**Comment by bcomnes** [comment]
Agreed, app should launch even if payment providers are not configured.

**Comment by TibebeJS** [comment]
Facing  the same issue when trying to run/see it locally.
Have tried `export PAYMENT_PAYPAL_ENABLED=true`, `export PAYMENT_PAYPAL_ENABLED=1`, etc.. with no luck.
Setting up payment should definitely be optional.
